tests/test_corner_flow_2d.py

```diff
@@ -211,29 +213,29 @@ def _get_velocity_gradient(point):
                     directions.append(bingham_vectors)
 
                 # Check that we start below the default cpo visualisation threshold.
-                assert misorient_indices[0] < 0.33
-                halfway = round(n_timesteps / 2)
+                # assert misorient_indices[0] < 0.33
+                # halfway = round(n_timesteps / 2)
                 # TODO: Use np.isclose
-                match z_exit:
-                    case -0.1:
-                        assert misorient_indices[halfway] > 0.75
-                        assert misorient_angles[halfway] > 17.5
-                        assert misorient_indices[-1] > 0.8
-                        assert misorient_angles[-1] < 17.5
-                    case -0.3:
-                        assert misorient_indices[halfway] > 0.625
-                        assert misorient_angles[halfway] > 20
-                        assert misorient_indices[-1] > 0.75
-                        assert misorient_angles[-1] < 20
-                    case -0.54:
-                        assert misorient_indices[halfway] > 0.575
-                        assert misorient_angles[halfway] > 22
-                        assert misorient_indices[-1] > 0.7
-                        assert misorient_angles[-1] < 22
-                    case -0.78:
-                        # TODO: Results from this pathline are a bit weird.
-                        # No tests for now...
-                        pass
+                # match z_exit:
+                #     case -0.1:
+                #         assert misorient_indices[halfway] > 0.75
+                #         assert misorient_angles[halfway] > 17.5
+                #         assert misorient_indices[-1] > 0.8
+                #         assert misorient_angles[-1] < 17.5
+                #     case -0.3:
+                #         assert misorient_indices[halfway] > 0.65
+                #         assert misorient_angles[halfway] > 20
+                #         assert misorient_indices[-1] > 0.75
+                #         assert misorient_angles[-1] < 20
+                #     case -0.54:
+                #         assert misorient_indices[halfway] > 0.575
+                #         assert misorient_angles[halfway] > 22
+                #         assert misorient_indices[-1] > 0.7
+                #         assert misorient_angles[-1] < 22
+                #     case -0.78:
+                #         # TODO: Results from this pathline are a bit weird.
+                #         # No tests for now...
+                #         pass
```

Comment:
Why did you comment all the `assert` statements?

Reply:
Becuase of the RNG seed, the `angles`, `indices`, etc. should now be reproducible, so I'm going to replace those somewhat arbitrary assserts with some `np.testing.assert_allclose` or statements instead.